### DISCUSSION TITLE
i cant use fabric 1.18

### DISCUSSION THREAD

**Discussion by vzixx** [FabricMC/fabric-api#2051] [Player support]
whenever i load up fabric, it shows the error message below. (i have tried both 1.18.2 versions and 1.18.1)[
![git](https://user-images.githubusercontent.com/101080564/156965189-b30edd91-0851-42d6-b14d-22e8956f625f.PNG)

**Comment by Linguardium** [comment]
you are using a 1.16.2+ version of fabric api instead of a 1.18.1 version

**Comment by chillytwastaken** [comment]
You are using the wrong version of Fabric API.

**Reply by chillytwastaken** [reply]
> You are using the wrong version of Fabric API.

Look at the "a potential solution has been determined" next time and try and figure it out.
